docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -178,6 +178,12 @@ include::partial$selenium-properties.adoc[]
 include::partial$selenium-grid-properties.adoc[]
 |screenshot.on-failure.debug-modes
 include::partial$proxy-properties.adoc[]
+
+|web.driver.CHROME.command-line-arguments
+|https://peter.sh/experiments/chromium-command-line-switches/[Chromium Command Line Switches]
+|empty
+|Space separated list of Chromium Command Line Switches
```

Comment:
we should either make this statement as much generic as possible (since we support not Chrome only), or it's needed to add similar description for all supported browsers

Reply:
Added description for other supported browsers